### PR TITLE
Add UCSCXenaShiny

### PR DESCRIPTION
**Pull request by ShixiangWang**
Checklist
- [x] Title of this PR is meaningful: e.g. "Adding my_nifty_package", not "updated meta.yaml".
- [x] License file is packaged (see [here](https://github.com/conda-forge/staged-recipes/blob/master/recipes/example/meta.yaml#L57-L66) for an example).
- [x] Source is from official source.
- [x] Package does not vendor other packages. (If a package uses the source of another package, they should be separate packages or the licenses of all packages need to be packaged).
- [x] If static libraries are linked in, the license of the static library is packaged.
- [x] Build number is 0.
- [x] A tarball (`url`) rather than a repo (e.g. `git_url`) is used in your recipe (see [here](https://conda-forge.org/docs/maintainer/adding_pkgs.html#build-from-tarballs-not-repos) for more details).
- [x] GitHub users listed in the maintainer section have posted a comment confirming they are willing to be listed there.
- [x] When in trouble, please check our [knowledge base documentation](https://conda-forge.org/docs/maintainer/knowledge_base.html) before pinging a team.

### CODE REVIEW

**Comment by ShixiangWang** [comment]
@conda-forge-admin, please ping team

The build failed due to colordiff，i dont know why？any suggestion？

**Comment by ShixiangWang** [comment]
@conda-forge/staged-recipes Ready for review!

**Comment by ShixiangWang** [comment]
@conda-forge-admin, please ping team

**Comment by ShixiangWang** [comment]
[@conda-forge/help-r](https://github.com/orgs/conda-forge/teams/help-r) ready for review.

**Comment by ShixiangWang** [comment]
Dear @dpryan79, could you help review and approve this PR? :)

File: recipes/ucscxenashiny/meta.yaml [hunk 1]
```diff
@@ -0,0 +1,103 @@
...
+    - {{ cran_mirror }}/src/contrib/UCSCXenaShiny_{{ version }}.tar.gz
+    - {{ cran_mirror }}/src/contrib/Archive/UCSCXenaShiny/UCSCXenaShiny_{{ version }}.tar.gz
+  sha256: 0669d9a0a1ecf117a668818c501addada6750df7db3619c9c9caa2c3d28f7648
+
+build:
+  merge_build_host: True  # [win]
+  # If this is a new build for the same version, increment the build number.
+  number: 0
+  # no skip
+
```
Comment (dbast) [inline recipes/ucscxenashiny/meta.yaml:21]: Can this be noarch as the others?

**Comment by dbast** [comment]
the recipes look fine except:
* r recipe folders are normally prefixed with "r-".
* One of the recipes looks like it could be noarch: generic like the others
* the recipes contain lots of commented lines ... either remove those manually or use https://github.com/bgruening/conda_r_skeleton_helper to regenerate the recipes (this also automatically does the folder prefixing and noarch)

**Comment by ShixiangWang** [comment]
@dbast Thanks for your comments. I have renamed the directories and removed comments. The linter suggested me to remove the `noarch` in `r-ucscxenashiny` as I added some required libraries only available in Linux. The two libraries are required in Linux to enable full features of this package.

```
  host:
    - gmp  # [linux]
    - mpfr  # [linux]
```

**Pull request merged**